pkgs/development/compilers/cudatoolkit/hooks/nvcc-setup-hook.sh

```diff
@@ -2,4 +2,4 @@
 
 # CMake's enable_language(CUDA) runs a compiler test and it doesn't account for
 # CUDAToolkit_ROOT. We have to help it locate libcudart
-export NVCC_APPEND_FLAGS+=" -L@cudartRoot@/lib -I@cudartRoot@/include"
+export NVCC_APPEND_FLAGS+=" -L@cudartRootLib@/lib -L@cudartRootStatic@/lib -I@cudartRootInclude@/include"
```

Comment:
In future we maybe should find a way to locate the cudart from buildInputs using setup hooks rather than using template rendering?.. The current set up pushes static outputs into build closure regardless of whether it's used. Outside the PR scope though

Reply:
Tracked in https://github.com/NixOS/nixpkgs/issues/248952